### PR TITLE
Display Toast Message Bottom on Mobile

### PR DESCRIPTION
**Pull request by vahidbazzaz**
This PR fixes displaying the toast message on the bottom in mobile view and on top in desktop view.

Follow up: https://github.com/boxwise/boxtribute/pull/591

### CODE REVIEW

**Review by pylipp — CHANGES_REQUESTED** [review]
@vahidbazzaz I tested it on a 1366px screen and the toast still shows at the bottom, how did you test it?

Anyways I think we shouldn't spend that much more time since the task is not prioritized.

**Comment by vahidbazzaz** [comment]
> @vahidbazzaz I tested it on a 1366px screen and the toast still shows at the bottom, how did you test it?
> 
> Anyways I think we shouldn't spend that much more time since the task is not prioritized.

@pylipp Thanks for checking that. I have checked it again and it seems to be working fine. If the screen size exceeds 1070px, the toast will appear at the top. Would you mind checking it out again when you have time?

**Comment by aerinsol** [comment]
@vahidbazzaz @HaGuesto @vahidbazzaz this feature is now live, did it get replaced by a different PR?

**Comment by vahidbazzaz** [comment]
> @vahidbazzaz @HaGuesto @vahidbazzaz this feature is now live, did it get replaced by a different PR?

@aerinsol This PR adds toast to the top on desktop, and to the bottom when below 1070 pixels. It complements Hans's previous [PR](https://github.com/boxwise/boxtribute/pull/591) that put all toast messages at the bottom. However, since this PR is not urgent, it has not been merged yet. Also needs to be reviewed.

File: react/src/utils/hooks.ts [hunk 1]
```diff
@@ -11,19 +16,25 @@ export interface INotificationProps extends UseToastOptions {
...
+    setPosition(isSmallScreen ? "bottom" : "top");
+  }, [isSmallScreen]);
 
   const createToast = useCallback(
     ({ message, type, ...props }: INotificationProps) =>
       toast({
         duration: 5000,
         isClosable: true,
-        position: "bottom",
+        position: position,
```
Comment (pylipp) [inline react/src/utils/hooks.ts:31]: I rebased on latest master and the project does not build anymore. In this line I get an eslint complaint `Expected property shorthand  object-shorthand`

**Comment by pylipp** [comment]
> To test, please follow these instructions:
> 
> Open the box view on a large screen (wider than 1070px) and click the Scrap or Lost button. The notification should appear on top. If you follow the same steps on a smaller screen or on a mobile device, you should see notification at the bottom of the screen

I can confirm that it works functionally, just an eslint complaint to be resolved after rebasing on master.

**Comment by HaGuesto** [comment]
@pylipp I fixed the eslint error. Ready to merge.

**Review by pylipp — APPROVED** [review]

**Pull request merged**